// @pnp/spfx-controls-react/lib/controls/filePicker/OneDriveFilesTab/OneDriveFilesTab.module.scss.js
/* tslint:disable */
require("./OneDriveFilesTab.module.css");
var styles = {
    filePicker: 'filePicker_2f4606d3',
    tabsContainer: 'tabsContainer_2f4606d3',
    tabContainer: 'tabContainer_2f4606d3',
    nav: 'nav_2f4606d3',
    tabHeaderContainer: 'tabHeaderContainer_2f4606d3',
    breadcrumbNav: 'breadcrumbNav_2f4606d3',
    breadcrumbNavItem: 'breadcrumbNavItem_2f4606d3',
    tab: 'tab_2f4606d3',
    tabOffset: 'tabOffset_2f4606d3',
    tabFiles: 'tabFiles_2f4606d3',
    scrollablePaneWrapper: 'scrollablePaneWrapper_2f4606d3',
    actionButtonsContainer: 'actionButtonsContainer_2f4606d3',
    actionButtons: 'actionButtons_2f4606d3',
    actionButton: 'actionButton_2f4606d3',
    tabHeader: 'tabHeader_2f4606d3',
    selectedTab: 'selectedTab_2f4606d3',
    header: 'header_2f4606d3',
    xlg: 'xlg_2f4606d3',
    xxlg: 'xxlg_2f4606d3',
    xxxlg: 'xxxlg_2f4606d3',
    xxxxlg: 'xxxxlg_2f4606d3',
    itemPickerTopBar: 'itemPickerTopBar_2f4606d3',
    commandBarNoChevron: 'commandBarNoChevron_2f4606d3',
    fileItem: 'fileItem_2f4606d3',
    folderItem: 'folderItem_2f4606d3',
    fileTypeIcon: 'fileTypeIcon_2f4606d3',
    fileTypeIconIcon: 'fileTypeIconIcon_2f4606d3',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_2f4606d3',
    iconColumnHeader: 'iconColumnHeader_2f4606d3',
    fileRow: 'fileRow_2f4606d3',
    folderRow: 'folderRow_2f4606d3',
    emptyFolder: 'emptyFolder_2f4606d3',
    emptyFolderImage: 'emptyFolderImage_2f4606d3',
    emptyFolderImageTag: 'emptyFolderImageTag_2f4606d3',
    emptyFolderTitle: 'emptyFolderTitle_2f4606d3',
    emptyFolderSubText: 'emptyFolderSubText_2f4606d3',
    emptyFolderMobile: 'emptyFolderMobile_2f4606d3',
    emptyFolderPc: 'emptyFolderPc_2f4606d3',
    emptyFolderIcon: 'emptyFolderIcon_2f4606d3'
};
export default styles;
/* tslint:enable */
