// @pnp/spfx-controls-react/lib/controls/filePicker/controls/DocumentTile/DocumentTile.module.scss.js
/* tslint:disable */
require("./DocumentTile.module.css");
var styles = {
    filePicker: 'filePicker_ab0a39b7',
    tabsContainer: 'tabsContainer_ab0a39b7',
    tabContainer: 'tabContainer_ab0a39b7',
    nav: 'nav_ab0a39b7',
    tabHeaderContainer: 'tabHeaderContainer_ab0a39b7',
    breadcrumbNav: 'breadcrumbNav_ab0a39b7',
    breadcrumbNavItem: 'breadcrumbNavItem_ab0a39b7',
    tab: 'tab_ab0a39b7',
    tabOffset: 'tabOffset_ab0a39b7',
    tabFiles: 'tabFiles_ab0a39b7',
    scrollablePaneWrapper: 'scrollablePaneWrapper_ab0a39b7',
    actionButtonsContainer: 'actionButtonsContainer_ab0a39b7',
    actionButtons: 'actionButtons_ab0a39b7',
    actionButton: 'actionButton_ab0a39b7',
    tabHeader: 'tabHeader_ab0a39b7',
    selectedTab: 'selectedTab_ab0a39b7',
    header: 'header_ab0a39b7',
    xlg: 'xlg_ab0a39b7',
    xxlg: 'xxlg_ab0a39b7',
    xxxlg: 'xxxlg_ab0a39b7',
    xxxxlg: 'xxxxlg_ab0a39b7',
    itemPickerTopBar: 'itemPickerTopBar_ab0a39b7',
    commandBarNoChevron: 'commandBarNoChevron_ab0a39b7',
    fileItem: 'fileItem_ab0a39b7',
    folderItem: 'folderItem_ab0a39b7',
    fileTypeIcon: 'fileTypeIcon_ab0a39b7',
    fileTypeIconIcon: 'fileTypeIconIcon_ab0a39b7',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_ab0a39b7',
    iconColumnHeader: 'iconColumnHeader_ab0a39b7',
    fileRow: 'fileRow_ab0a39b7',
    folderRow: 'folderRow_ab0a39b7',
    emptyFolder: 'emptyFolder_ab0a39b7',
    emptyFolderImage: 'emptyFolderImage_ab0a39b7',
    emptyFolderImageTag: 'emptyFolderImageTag_ab0a39b7',
    emptyFolderTitle: 'emptyFolderTitle_ab0a39b7',
    emptyFolderSubText: 'emptyFolderSubText_ab0a39b7',
    emptyFolderMobile: 'emptyFolderMobile_ab0a39b7',
    emptyFolderPc: 'emptyFolderPc_ab0a39b7',
    emptyFolderIcon: 'emptyFolderIcon_ab0a39b7',
    aboveNameplate: 'aboveNameplate_ab0a39b7',
    tileContent: 'tileContent_ab0a39b7',
    foreground: 'foreground_ab0a39b7',
    grid: 'grid_ab0a39b7',
    cellContent: 'cellContent_ab0a39b7',
    content: 'content_ab0a39b7',
    tile: 'tile_ab0a39b7',
    invokable: 'invokable_ab0a39b7',
    namePlate: 'namePlate_ab0a39b7',
    nameplate: 'nameplate_ab0a39b7',
    name: 'name_ab0a39b7',
    isLarge: 'isLarge_ab0a39b7',
    activity: 'activity_ab0a39b7',
    signalField: 'signalField_ab0a39b7',
    compact: 'compact_ab0a39b7',
    signalFieldValue: 'signalFieldValue_ab0a39b7',
    folderList: 'folderList_ab0a39b7',
    link: 'link_ab0a39b7',
    cell: 'cell_ab0a39b7',
    listCell: 'listCell_ab0a39b7',
    label: 'label_ab0a39b7',
    check: 'check_ab0a39b7',
    selectable: 'selectable_ab0a39b7',
    showCheck: 'showCheck_ab0a39b7',
    selected: 'selected_ab0a39b7',
    dialogMainOverride: 'dialogMainOverride_ab0a39b7',
    itemTileBlankCover: 'itemTileBlankCover_ab0a39b7',
    metadata: 'metadata_ab0a39b7',
    isSmall: 'isSmall_ab0a39b7',
    listPage: 'listPage_ab0a39b7',
    odItemTile2FolderCover: 'odItemTile2FolderCover_ab0a39b7',
    standaloneListBreadcrumb: 'standaloneListBreadcrumb_ab0a39b7',
    odItemTile2Image: 'odItemTile2Image_ab0a39b7',
    odImageFrame2: 'odImageFrame2_ab0a39b7',
    odImageFrame2Image: 'odImageFrame2Image_ab0a39b7',
    odImageFrame: 'odImageFrame_ab0a39b7',
    odImageStack: 'odImageStack_ab0a39b7',
    odImageStackTile: 'odImageStackTile_ab0a39b7',
    odImageTile: 'odImageTile_ab0a39b7',
    odImageTileBackground: 'odImageTileBackground_ab0a39b7',
    odImageTileImage: 'odImageTileImage_ab0a39b7',
    odItemTile2SmallIcon: 'odItemTile2SmallIcon_ab0a39b7'
};
export default styles;
/* tslint:enable */
