// @pnp/spfx-controls-react/lib/controls/filePicker/WebSearchTab/WebSearchTab.module.scss.js
/* tslint:disable */
require("./WebSearchTab.module.css");
var styles = {
    filePicker: 'filePicker_d5ec7c40',
    tabsContainer: 'tabsContainer_d5ec7c40',
    tabContainer: 'tabContainer_d5ec7c40',
    nav: 'nav_d5ec7c40',
    tabHeaderContainer: 'tabHeaderContainer_d5ec7c40',
    breadcrumbNav: 'breadcrumbNav_d5ec7c40',
    breadcrumbNavItem: 'breadcrumbNavItem_d5ec7c40',
    tab: 'tab_d5ec7c40',
    tabOffset: 'tabOffset_d5ec7c40',
    tabFiles: 'tabFiles_d5ec7c40',
    scrollablePaneWrapper: 'scrollablePaneWrapper_d5ec7c40',
    actionButtonsContainer: 'actionButtonsContainer_d5ec7c40',
    actionButtons: 'actionButtons_d5ec7c40',
    actionButton: 'actionButton_d5ec7c40',
    tabHeader: 'tabHeader_d5ec7c40',
    selectedTab: 'selectedTab_d5ec7c40',
    header: 'header_d5ec7c40',
    xlg: 'xlg_d5ec7c40',
    xxlg: 'xxlg_d5ec7c40',
    xxxlg: 'xxxlg_d5ec7c40',
    xxxxlg: 'xxxxlg_d5ec7c40',
    itemPickerTopBar: 'itemPickerTopBar_d5ec7c40',
    commandBarNoChevron: 'commandBarNoChevron_d5ec7c40',
    fileItem: 'fileItem_d5ec7c40',
    folderItem: 'folderItem_d5ec7c40',
    fileTypeIcon: 'fileTypeIcon_d5ec7c40',
    fileTypeIconIcon: 'fileTypeIconIcon_d5ec7c40',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_d5ec7c40',
    iconColumnHeader: 'iconColumnHeader_d5ec7c40',
    fileRow: 'fileRow_d5ec7c40',
    folderRow: 'folderRow_d5ec7c40',
    emptyFolder: 'emptyFolder_d5ec7c40',
    emptyFolderImage: 'emptyFolderImage_d5ec7c40',
    emptyFolderImageTag: 'emptyFolderImageTag_d5ec7c40',
    emptyFolderTitle: 'emptyFolderTitle_d5ec7c40',
    emptyFolderSubText: 'emptyFolderSubText_d5ec7c40',
    emptyFolderMobile: 'emptyFolderMobile_d5ec7c40',
    emptyFolderPc: 'emptyFolderPc_d5ec7c40',
    emptyFolderIcon: 'emptyFolderIcon_d5ec7c40',
    searchBoxContainer: 'searchBoxContainer_d5ec7c40',
    searchBoxMedium: 'searchBoxMedium_d5ec7c40',
    searchBox: 'searchBox_d5ec7c40',
    dropdownContainer: 'dropdownContainer_d5ec7c40',
    filterDropdown: 'filterDropdown_d5ec7c40',
    filePickerFolderCardGrid: 'filePickerFolderCardGrid_d5ec7c40',
    filePickerFolderCardTile: 'filePickerFolderCardTile_d5ec7c40',
    filePickerFolderCardPadder: 'filePickerFolderCardPadder_d5ec7c40',
    filePickerFolderCardSizer: 'filePickerFolderCardSizer_d5ec7c40',
    filePickerFolderCardImage: 'filePickerFolderCardImage_d5ec7c40',
    filePickerFolderCardLabel: 'filePickerFolderCardLabel_d5ec7c40',
    bingGridListCell: 'bingGridListCell_d5ec7c40',
    bingTile: 'bingTile_d5ec7c40',
    bingTileContent: 'bingTileContent_d5ec7c40',
    bingTileFrame: 'bingTileFrame_d5ec7c40',
    bingTileCheckCircle: 'bingTileCheckCircle_d5ec7c40',
    bingTileNamePlate: 'bingTileNamePlate_d5ec7c40',
    isSelected: 'isSelected_d5ec7c40',
    copyrightLabel: 'copyrightLabel_d5ec7c40',
    noResultLabel: 'noResultLabel_d5ec7c40',
    bingTileThumbnail: 'bingTileThumbnail_d5ec7c40',
    bingGrildList: 'bingGrildList_d5ec7c40'
};
export default styles;
/* tslint:enable */
