// @pnp/spfx-controls-react/lib/controls/filePicker/FilePicker.module.scss.js
/* tslint:disable */
require("./FilePicker.module.css");
var styles = {
    filePicker: 'filePicker_01b16b2e',
    tabsContainer: 'tabsContainer_01b16b2e',
    tabContainer: 'tabContainer_01b16b2e',
    nav: 'nav_01b16b2e',
    tabHeaderContainer: 'tabHeaderContainer_01b16b2e',
    breadcrumbNav: 'breadcrumbNav_01b16b2e',
    breadcrumbNavItem: 'breadcrumbNavItem_01b16b2e',
    tab: 'tab_01b16b2e',
    tabOffset: 'tabOffset_01b16b2e',
    tabFiles: 'tabFiles_01b16b2e',
    scrollablePaneWrapper: 'scrollablePaneWrapper_01b16b2e',
    actionButtonsContainer: 'actionButtonsContainer_01b16b2e',
    actionButtons: 'actionButtons_01b16b2e',
    actionButton: 'actionButton_01b16b2e',
    tabHeader: 'tabHeader_01b16b2e',
    selectedTab: 'selectedTab_01b16b2e',
    header: 'header_01b16b2e',
    xlg: 'xlg_01b16b2e',
    xxlg: 'xxlg_01b16b2e',
    xxxlg: 'xxxlg_01b16b2e',
    xxxxlg: 'xxxxlg_01b16b2e',
    itemPickerTopBar: 'itemPickerTopBar_01b16b2e',
    commandBarNoChevron: 'commandBarNoChevron_01b16b2e',
    fileItem: 'fileItem_01b16b2e',
    folderItem: 'folderItem_01b16b2e',
    fileTypeIcon: 'fileTypeIcon_01b16b2e',
    fileTypeIconIcon: 'fileTypeIconIcon_01b16b2e',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_01b16b2e',
    iconColumnHeader: 'iconColumnHeader_01b16b2e',
    fileRow: 'fileRow_01b16b2e',
    folderRow: 'folderRow_01b16b2e',
    emptyFolder: 'emptyFolder_01b16b2e',
    emptyFolderImage: 'emptyFolderImage_01b16b2e',
    emptyFolderImageTag: 'emptyFolderImageTag_01b16b2e',
    emptyFolderTitle: 'emptyFolderTitle_01b16b2e',
    emptyFolderSubText: 'emptyFolderSubText_01b16b2e',
    emptyFolderMobile: 'emptyFolderMobile_01b16b2e',
    emptyFolderPc: 'emptyFolderPc_01b16b2e',
    emptyFolderIcon: 'emptyFolderIcon_01b16b2e'
};
export default styles;
/* tslint:enable */
