// @pnp/spfx-controls-react/lib/controls/filePicker/LinkFilePickerTab/LinkFilePickerTab.module.scss.js
/* tslint:disable */
require("./LinkFilePickerTab.module.css");
var styles = {
    filePicker: 'filePicker_75566c17',
    tabsContainer: 'tabsContainer_75566c17',
    tabContainer: 'tabContainer_75566c17',
    nav: 'nav_75566c17',
    tabHeaderContainer: 'tabHeaderContainer_75566c17',
    breadcrumbNav: 'breadcrumbNav_75566c17',
    breadcrumbNavItem: 'breadcrumbNavItem_75566c17',
    tab: 'tab_75566c17',
    tabOffset: 'tabOffset_75566c17',
    tabFiles: 'tabFiles_75566c17',
    scrollablePaneWrapper: 'scrollablePaneWrapper_75566c17',
    actionButtonsContainer: 'actionButtonsContainer_75566c17',
    actionButtons: 'actionButtons_75566c17',
    actionButton: 'actionButton_75566c17',
    tabHeader: 'tabHeader_75566c17',
    selectedTab: 'selectedTab_75566c17',
    header: 'header_75566c17',
    xlg: 'xlg_75566c17',
    xxlg: 'xxlg_75566c17',
    xxxlg: 'xxxlg_75566c17',
    xxxxlg: 'xxxxlg_75566c17',
    itemPickerTopBar: 'itemPickerTopBar_75566c17',
    commandBarNoChevron: 'commandBarNoChevron_75566c17',
    fileItem: 'fileItem_75566c17',
    folderItem: 'folderItem_75566c17',
    fileTypeIcon: 'fileTypeIcon_75566c17',
    fileTypeIconIcon: 'fileTypeIconIcon_75566c17',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_75566c17',
    iconColumnHeader: 'iconColumnHeader_75566c17',
    fileRow: 'fileRow_75566c17',
    folderRow: 'folderRow_75566c17',
    emptyFolder: 'emptyFolder_75566c17',
    emptyFolderImage: 'emptyFolderImage_75566c17',
    emptyFolderImageTag: 'emptyFolderImageTag_75566c17',
    emptyFolderTitle: 'emptyFolderTitle_75566c17',
    emptyFolderSubText: 'emptyFolderSubText_75566c17',
    emptyFolderMobile: 'emptyFolderMobile_75566c17',
    emptyFolderPc: 'emptyFolderPc_75566c17',
    emptyFolderIcon: 'emptyFolderIcon_75566c17',
    linkTextField: 'linkTextField_75566c17',
    localTabSinglePreview: 'localTabSinglePreview_75566c17',
    localTabSinglePreviewImage: 'localTabSinglePreviewImage_75566c17'
};
export default styles;
/* tslint:enable */
