// @pnp/spfx-controls-react/lib/controls/filePicker/SiteFilePickerTab/SiteFilePickerTab.module.scss.js
/* tslint:disable */
require("./SiteFilePickerTab.module.css");
var styles = {
    filePicker: 'filePicker_dccc557c',
    tabsContainer: 'tabsContainer_dccc557c',
    tabContainer: 'tabContainer_dccc557c',
    nav: 'nav_dccc557c',
    tabHeaderContainer: 'tabHeaderContainer_dccc557c',
    breadcrumbNav: 'breadcrumbNav_dccc557c',
    breadcrumbNavItem: 'breadcrumbNavItem_dccc557c',
    tab: 'tab_dccc557c',
    tabOffset: 'tabOffset_dccc557c',
    tabFiles: 'tabFiles_dccc557c',
    scrollablePaneWrapper: 'scrollablePaneWrapper_dccc557c',
    actionButtonsContainer: 'actionButtonsContainer_dccc557c',
    actionButtons: 'actionButtons_dccc557c',
    actionButton: 'actionButton_dccc557c',
    tabHeader: 'tabHeader_dccc557c',
    selectedTab: 'selectedTab_dccc557c',
    header: 'header_dccc557c',
    xlg: 'xlg_dccc557c',
    xxlg: 'xxlg_dccc557c',
    xxxlg: 'xxxlg_dccc557c',
    xxxxlg: 'xxxxlg_dccc557c',
    itemPickerTopBar: 'itemPickerTopBar_dccc557c',
    commandBarNoChevron: 'commandBarNoChevron_dccc557c',
    fileItem: 'fileItem_dccc557c',
    folderItem: 'folderItem_dccc557c',
    fileTypeIcon: 'fileTypeIcon_dccc557c',
    fileTypeIconIcon: 'fileTypeIconIcon_dccc557c',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_dccc557c',
    iconColumnHeader: 'iconColumnHeader_dccc557c',
    fileRow: 'fileRow_dccc557c',
    folderRow: 'folderRow_dccc557c',
    emptyFolder: 'emptyFolder_dccc557c',
    emptyFolderImage: 'emptyFolderImage_dccc557c',
    emptyFolderImageTag: 'emptyFolderImageTag_dccc557c',
    emptyFolderTitle: 'emptyFolderTitle_dccc557c',
    emptyFolderSubText: 'emptyFolderSubText_dccc557c',
    emptyFolderMobile: 'emptyFolderMobile_dccc557c',
    emptyFolderPc: 'emptyFolderPc_dccc557c',
    emptyFolderIcon: 'emptyFolderIcon_dccc557c'
};
export default styles;
/* tslint:enable */
