// @pnp/spfx-controls-react/lib/controls/filePicker/StockImagesTab/StockImages.module.scss.js
/* tslint:disable */
require("./StockImages.module.css");
var styles = {
    filePicker: 'filePicker_1f6d256e',
    tabsContainer: 'tabsContainer_1f6d256e',
    tabContainer: 'tabContainer_1f6d256e',
    nav: 'nav_1f6d256e',
    tabHeaderContainer: 'tabHeaderContainer_1f6d256e',
    breadcrumbNav: 'breadcrumbNav_1f6d256e',
    breadcrumbNavItem: 'breadcrumbNavItem_1f6d256e',
    tab: 'tab_1f6d256e',
    tabOffset: 'tabOffset_1f6d256e',
    tabFiles: 'tabFiles_1f6d256e',
    scrollablePaneWrapper: 'scrollablePaneWrapper_1f6d256e',
    actionButtonsContainer: 'actionButtonsContainer_1f6d256e',
    actionButtons: 'actionButtons_1f6d256e',
    actionButton: 'actionButton_1f6d256e',
    tabHeader: 'tabHeader_1f6d256e',
    selectedTab: 'selectedTab_1f6d256e',
    header: 'header_1f6d256e',
    xlg: 'xlg_1f6d256e',
    xxlg: 'xxlg_1f6d256e',
    xxxlg: 'xxxlg_1f6d256e',
    xxxxlg: 'xxxxlg_1f6d256e',
    itemPickerTopBar: 'itemPickerTopBar_1f6d256e',
    commandBarNoChevron: 'commandBarNoChevron_1f6d256e',
    fileItem: 'fileItem_1f6d256e',
    folderItem: 'folderItem_1f6d256e',
    fileTypeIcon: 'fileTypeIcon_1f6d256e',
    fileTypeIconIcon: 'fileTypeIconIcon_1f6d256e',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_1f6d256e',
    iconColumnHeader: 'iconColumnHeader_1f6d256e',
    fileRow: 'fileRow_1f6d256e',
    folderRow: 'folderRow_1f6d256e',
    emptyFolder: 'emptyFolder_1f6d256e',
    emptyFolderImage: 'emptyFolderImage_1f6d256e',
    emptyFolderImageTag: 'emptyFolderImageTag_1f6d256e',
    emptyFolderTitle: 'emptyFolderTitle_1f6d256e',
    emptyFolderSubText: 'emptyFolderSubText_1f6d256e',
    emptyFolderMobile: 'emptyFolderMobile_1f6d256e',
    emptyFolderPc: 'emptyFolderPc_1f6d256e',
    emptyFolderIcon: 'emptyFolderIcon_1f6d256e',
    stockImagesPickerContainer: 'stockImagesPickerContainer_1f6d256e',
    stockImagesPicker: 'stockImagesPicker_1f6d256e'
};
export default styles;
/* tslint:enable */
