// @pnp/spfx-controls-react/lib/controls/filePicker/controls/FileBrowser/FileBrowser.module.scss.js
/* tslint:disable */
require("./FileBrowser.module.css");
var styles = {
    filePicker: 'filePicker_93587cff',
    tabsContainer: 'tabsContainer_93587cff',
    tabContainer: 'tabContainer_93587cff',
    nav: 'nav_93587cff',
    tabHeaderContainer: 'tabHeaderContainer_93587cff',
    breadcrumbNav: 'breadcrumbNav_93587cff',
    breadcrumbNavItem: 'breadcrumbNavItem_93587cff',
    tab: 'tab_93587cff',
    tabOffset: 'tabOffset_93587cff',
    tabFiles: 'tabFiles_93587cff',
    scrollablePaneWrapper: 'scrollablePaneWrapper_93587cff',
    actionButtonsContainer: 'actionButtonsContainer_93587cff',
    actionButtons: 'actionButtons_93587cff',
    actionButton: 'actionButton_93587cff',
    tabHeader: 'tabHeader_93587cff',
    selectedTab: 'selectedTab_93587cff',
    header: 'header_93587cff',
    xlg: 'xlg_93587cff',
    xxlg: 'xxlg_93587cff',
    xxxlg: 'xxxlg_93587cff',
    xxxxlg: 'xxxxlg_93587cff',
    itemPickerTopBar: 'itemPickerTopBar_93587cff',
    commandBarNoChevron: 'commandBarNoChevron_93587cff',
    fileItem: 'fileItem_93587cff',
    folderItem: 'folderItem_93587cff',
    fileTypeIcon: 'fileTypeIcon_93587cff',
    fileTypeIconIcon: 'fileTypeIconIcon_93587cff',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_93587cff',
    iconColumnHeader: 'iconColumnHeader_93587cff',
    fileRow: 'fileRow_93587cff',
    folderRow: 'folderRow_93587cff',
    emptyFolder: 'emptyFolder_93587cff',
    emptyFolderImage: 'emptyFolderImage_93587cff',
    emptyFolderImageTag: 'emptyFolderImageTag_93587cff',
    emptyFolderTitle: 'emptyFolderTitle_93587cff',
    emptyFolderSubText: 'emptyFolderSubText_93587cff',
    emptyFolderMobile: 'emptyFolderMobile_93587cff',
    emptyFolderPc: 'emptyFolderPc_93587cff',
    emptyFolderIcon: 'emptyFolderIcon_93587cff'
};
export default styles;
/* tslint:enable */
