// @pnp/spfx-controls-react/lib/controls/filePicker/controls/FolderTile/FolderTile.module.scss.js
/* tslint:disable */
require("./FolderTile.module.css");
var styles = {
    filePicker: 'filePicker_bc8aec8c',
    tabsContainer: 'tabsContainer_bc8aec8c',
    tabContainer: 'tabContainer_bc8aec8c',
    nav: 'nav_bc8aec8c',
    tabHeaderContainer: 'tabHeaderContainer_bc8aec8c',
    breadcrumbNav: 'breadcrumbNav_bc8aec8c',
    breadcrumbNavItem: 'breadcrumbNavItem_bc8aec8c',
    tab: 'tab_bc8aec8c',
    tabOffset: 'tabOffset_bc8aec8c',
    tabFiles: 'tabFiles_bc8aec8c',
    scrollablePaneWrapper: 'scrollablePaneWrapper_bc8aec8c',
    actionButtonsContainer: 'actionButtonsContainer_bc8aec8c',
    actionButtons: 'actionButtons_bc8aec8c',
    actionButton: 'actionButton_bc8aec8c',
    tabHeader: 'tabHeader_bc8aec8c',
    selectedTab: 'selectedTab_bc8aec8c',
    header: 'header_bc8aec8c',
    xlg: 'xlg_bc8aec8c',
    xxlg: 'xxlg_bc8aec8c',
    xxxlg: 'xxxlg_bc8aec8c',
    xxxxlg: 'xxxxlg_bc8aec8c',
    itemPickerTopBar: 'itemPickerTopBar_bc8aec8c',
    commandBarNoChevron: 'commandBarNoChevron_bc8aec8c',
    fileItem: 'fileItem_bc8aec8c',
    folderItem: 'folderItem_bc8aec8c',
    fileTypeIcon: 'fileTypeIcon_bc8aec8c',
    fileTypeIconIcon: 'fileTypeIconIcon_bc8aec8c',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_bc8aec8c',
    iconColumnHeader: 'iconColumnHeader_bc8aec8c',
    fileRow: 'fileRow_bc8aec8c',
    folderRow: 'folderRow_bc8aec8c',
    emptyFolder: 'emptyFolder_bc8aec8c',
    emptyFolderImage: 'emptyFolderImage_bc8aec8c',
    emptyFolderImageTag: 'emptyFolderImageTag_bc8aec8c',
    emptyFolderTitle: 'emptyFolderTitle_bc8aec8c',
    emptyFolderSubText: 'emptyFolderSubText_bc8aec8c',
    emptyFolderMobile: 'emptyFolderMobile_bc8aec8c',
    emptyFolderPc: 'emptyFolderPc_bc8aec8c',
    emptyFolderIcon: 'emptyFolderIcon_bc8aec8c',
    aboveNameplate: 'aboveNameplate_bc8aec8c',
    tileContent: 'tileContent_bc8aec8c',
    foreground: 'foreground_bc8aec8c',
    grid: 'grid_bc8aec8c',
    cellContent: 'cellContent_bc8aec8c',
    content: 'content_bc8aec8c',
    tile: 'tile_bc8aec8c',
    invokable: 'invokable_bc8aec8c',
    namePlate: 'namePlate_bc8aec8c',
    nameplate: 'nameplate_bc8aec8c',
    name: 'name_bc8aec8c',
    isLarge: 'isLarge_bc8aec8c',
    activity: 'activity_bc8aec8c',
    signalField: 'signalField_bc8aec8c',
    compact: 'compact_bc8aec8c',
    signalFieldValue: 'signalFieldValue_bc8aec8c',
    folderList: 'folderList_bc8aec8c',
    link: 'link_bc8aec8c',
    cell: 'cell_bc8aec8c',
    listCell: 'listCell_bc8aec8c',
    label: 'label_bc8aec8c',
    check: 'check_bc8aec8c',
    selectable: 'selectable_bc8aec8c',
    showCheck: 'showCheck_bc8aec8c',
    selected: 'selected_bc8aec8c',
    dialogMainOverride: 'dialogMainOverride_bc8aec8c',
    itemTileBlankCover: 'itemTileBlankCover_bc8aec8c',
    metadata: 'metadata_bc8aec8c',
    isSmall: 'isSmall_bc8aec8c',
    listPage: 'listPage_bc8aec8c',
    odItemTile2FolderCover: 'odItemTile2FolderCover_bc8aec8c',
    standaloneListBreadcrumb: 'standaloneListBreadcrumb_bc8aec8c',
    folderCover: 'folderCover_bc8aec8c',
    isSmall_88d29bd4: 'isSmall_88d29bd4_bc8aec8c',
    folderCoverBack: 'folderCoverBack_bc8aec8c',
    folderCoverFront: 'folderCoverFront_bc8aec8c',
    folderCoverFrame: 'folderCoverFrame_bc8aec8c',
    folderCoverContent: 'folderCoverContent_bc8aec8c'
};
export default styles;
/* tslint:enable */
