// @pnp/spfx-controls-react/lib/controls/filePicker/RecentFilesTab/RecentFilesTab.module.scss.js
/* tslint:disable */
require("./RecentFilesTab.module.css");
var styles = {
    filePicker: 'filePicker_a95647c6',
    tabsContainer: 'tabsContainer_a95647c6',
    tabContainer: 'tabContainer_a95647c6',
    nav: 'nav_a95647c6',
    tabHeaderContainer: 'tabHeaderContainer_a95647c6',
    breadcrumbNav: 'breadcrumbNav_a95647c6',
    breadcrumbNavItem: 'breadcrumbNavItem_a95647c6',
    tab: 'tab_a95647c6',
    tabOffset: 'tabOffset_a95647c6',
    tabFiles: 'tabFiles_a95647c6',
    scrollablePaneWrapper: 'scrollablePaneWrapper_a95647c6',
    actionButtonsContainer: 'actionButtonsContainer_a95647c6',
    actionButtons: 'actionButtons_a95647c6',
    actionButton: 'actionButton_a95647c6',
    tabHeader: 'tabHeader_a95647c6',
    selectedTab: 'selectedTab_a95647c6',
    header: 'header_a95647c6',
    xlg: 'xlg_a95647c6',
    xxlg: 'xxlg_a95647c6',
    xxxlg: 'xxxlg_a95647c6',
    xxxxlg: 'xxxxlg_a95647c6',
    itemPickerTopBar: 'itemPickerTopBar_a95647c6',
    commandBarNoChevron: 'commandBarNoChevron_a95647c6',
    fileItem: 'fileItem_a95647c6',
    folderItem: 'folderItem_a95647c6',
    fileTypeIcon: 'fileTypeIcon_a95647c6',
    fileTypeIconIcon: 'fileTypeIconIcon_a95647c6',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_a95647c6',
    iconColumnHeader: 'iconColumnHeader_a95647c6',
    fileRow: 'fileRow_a95647c6',
    folderRow: 'folderRow_a95647c6',
    emptyFolder: 'emptyFolder_a95647c6',
    emptyFolderImage: 'emptyFolderImage_a95647c6',
    emptyFolderImageTag: 'emptyFolderImageTag_a95647c6',
    emptyFolderTitle: 'emptyFolderTitle_a95647c6',
    emptyFolderSubText: 'emptyFolderSubText_a95647c6',
    emptyFolderMobile: 'emptyFolderMobile_a95647c6',
    emptyFolderPc: 'emptyFolderPc_a95647c6',
    emptyFolderIcon: 'emptyFolderIcon_a95647c6',
    gridListCell: 'gridListCell_a95647c6',
    cellRight: 'cellRight_a95647c6',
    cellLastRow: 'cellLastRow_a95647c6',
    itemTile: 'itemTile_a95647c6',
    isFile: 'isFile_a95647c6',
    isPhoto: 'isPhoto_a95647c6',
    isVideo: 'isVideo_a95647c6',
    hasThumbnail: 'hasThumbnail_a95647c6',
    itemTileContent: 'itemTileContent_a95647c6',
    itemTileDiscoverFileContainer: 'itemTileDiscoverFileContainer_a95647c6',
    itemTileFileContainer: 'itemTileFileContainer_a95647c6',
    itemTileFile: 'itemTileFile_a95647c6',
    itemTileThumbnail: 'itemTileThumbnail_a95647c6',
    image: 'image_a95647c6',
    itemTileCheckCircle: 'itemTileCheckCircle_a95647c6',
    isSelected: 'isSelected_a95647c6',
    itemTileNamePlate: 'itemTileNamePlate_a95647c6',
    itemTileName: 'itemTileName_a95647c6',
    itemTileChildCount: 'itemTileChildCount_a95647c6',
    itemTileSubText: 'itemTileSubText_a95647c6',
    recentGridList: 'recentGridList_a95647c6'
};
export default styles;
/* tslint:enable */
