// @pnp/spfx-controls-react/lib/controls/filePicker/controls/TilesList/TilesList.module.scss.js
/* tslint:disable */
require("./TilesList.module.css");
var styles = {
    filePicker: 'filePicker_5cbe0329',
    tabsContainer: 'tabsContainer_5cbe0329',
    tabContainer: 'tabContainer_5cbe0329',
    nav: 'nav_5cbe0329',
    tabHeaderContainer: 'tabHeaderContainer_5cbe0329',
    breadcrumbNav: 'breadcrumbNav_5cbe0329',
    breadcrumbNavItem: 'breadcrumbNavItem_5cbe0329',
    tab: 'tab_5cbe0329',
    tabOffset: 'tabOffset_5cbe0329',
    tabFiles: 'tabFiles_5cbe0329',
    scrollablePaneWrapper: 'scrollablePaneWrapper_5cbe0329',
    actionButtonsContainer: 'actionButtonsContainer_5cbe0329',
    actionButtons: 'actionButtons_5cbe0329',
    actionButton: 'actionButton_5cbe0329',
    tabHeader: 'tabHeader_5cbe0329',
    selectedTab: 'selectedTab_5cbe0329',
    header: 'header_5cbe0329',
    xlg: 'xlg_5cbe0329',
    xxlg: 'xxlg_5cbe0329',
    xxxlg: 'xxxlg_5cbe0329',
    xxxxlg: 'xxxxlg_5cbe0329',
    itemPickerTopBar: 'itemPickerTopBar_5cbe0329',
    commandBarNoChevron: 'commandBarNoChevron_5cbe0329',
    fileItem: 'fileItem_5cbe0329',
    folderItem: 'folderItem_5cbe0329',
    fileTypeIcon: 'fileTypeIcon_5cbe0329',
    fileTypeIconIcon: 'fileTypeIconIcon_5cbe0329',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_5cbe0329',
    iconColumnHeader: 'iconColumnHeader_5cbe0329',
    fileRow: 'fileRow_5cbe0329',
    folderRow: 'folderRow_5cbe0329',
    emptyFolder: 'emptyFolder_5cbe0329',
    emptyFolderImage: 'emptyFolderImage_5cbe0329',
    emptyFolderImageTag: 'emptyFolderImageTag_5cbe0329',
    emptyFolderTitle: 'emptyFolderTitle_5cbe0329',
    emptyFolderSubText: 'emptyFolderSubText_5cbe0329',
    emptyFolderMobile: 'emptyFolderMobile_5cbe0329',
    emptyFolderPc: 'emptyFolderPc_5cbe0329',
    emptyFolderIcon: 'emptyFolderIcon_5cbe0329',
    aboveNameplate: 'aboveNameplate_5cbe0329',
    tileContent: 'tileContent_5cbe0329',
    foreground: 'foreground_5cbe0329',
    grid: 'grid_5cbe0329',
    cellContent: 'cellContent_5cbe0329',
    content: 'content_5cbe0329',
    tile: 'tile_5cbe0329',
    invokable: 'invokable_5cbe0329',
    namePlate: 'namePlate_5cbe0329',
    nameplate: 'nameplate_5cbe0329',
    name: 'name_5cbe0329',
    isLarge: 'isLarge_5cbe0329',
    activity: 'activity_5cbe0329',
    signalField: 'signalField_5cbe0329',
    compact: 'compact_5cbe0329',
    signalFieldValue: 'signalFieldValue_5cbe0329',
    folderList: 'folderList_5cbe0329',
    link: 'link_5cbe0329',
    cell: 'cell_5cbe0329',
    listCell: 'listCell_5cbe0329',
    label: 'label_5cbe0329',
    check: 'check_5cbe0329',
    selectable: 'selectable_5cbe0329',
    showCheck: 'showCheck_5cbe0329',
    selected: 'selected_5cbe0329',
    dialogMainOverride: 'dialogMainOverride_5cbe0329',
    itemTileBlankCover: 'itemTileBlankCover_5cbe0329',
    metadata: 'metadata_5cbe0329',
    isSmall: 'isSmall_5cbe0329',
    listPage: 'listPage_5cbe0329',
    odItemTile2FolderCover: 'odItemTile2FolderCover_5cbe0329',
    standaloneListBreadcrumb: 'standaloneListBreadcrumb_5cbe0329'
};
export default styles;
/* tslint:enable */
